docs/docs/middleware_reference/url.md

```diff
@@ -1,11 +1,12 @@
 # URL
-Parses the HTTP request URL and sets the appropriate site configuration for current request. If the request URL is listed among the `extra` addresses a 301 redirect to the canonical URL is made. If there is no site with the requested URL a response with 404 HTTP status code is returned.
+Parses the HTTP request URL and sets the appropriate site configuration for current request. If the request URL is listed among the `extra` addresses a 301 redirect to the canonical URL is made. If there is no site with the requested URL a response with 404 HTTP status code is returned. If the `ssl` is set to `true` a 301 redirect to HTTPS is made.
```

Comment:
Could we make it a bit more descriptive please? So that it reads:

If the `ssl` option is set to `true` a 301 redirect to the same path over HTTPS is made.